### DISCUSSION TITLE
does pr-compliance-action leave too many comments?

### DISCUSSION THREAD

**Discussion by mtfoley** [open-sauced/open-sauced#1350] [Ideas] [answered]
For reference see: https://github.com/mtfoley/pr-compliance-action/issues/98

case in point #1348

**Comment by 0-vortex** [comment]
Can't say yes 💯

**Comment by mtfoley** [comment]
echoed in the action repo, but
> Perhaps instead of using comments, the action could initiate/update a pull request review. I think that they're a little bit less likely to be mixed up with something else done by github actions. I'm not sure if the action would have sufficient permissions to dismiss a review, but it seems like updating the body would be enough.
https://octokit.github.io/rest.js/v18#pulls-create-review
https://octokit.github.io/rest.js/v18#pulls-update-review

**Comment by mtfoley** [comment, marked as answer]
This is addressed in #1352
